Gemfile

```diff
@@ -0,0 +1,15 @@
+ruby '2.2.2'
+
+gem 'activesupport', '~> 4.2.2'
+gem 'elasticsearch-model'
```

Comment:
I would like to pin this but the last tag is from May 2016, so i'd need to double-check we weren't relying on any features developed in the meantime (there has been on-going development). I'll try to pin the gems below this though.